index.html

```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
           <li>Terminal</li>
         </ul>
       </section>
+      <section class="grid work">
+        <div class="container handles">
+          <img src="media/icons/github.svg" alt="github">
+          <img src="media/icons/linkedin.svg" alt="linkedin">
+          <img src="media/icons/peace.svg" alt="peace">
+          <img src="media/icons/twitter.svg" alt="twitter">
+          <img src="media/icons/m.svg" alt="m">
+        </div>
```

Comment:
- [ ] In order to make use of semantic tags, this list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)

Reply:
Not full resolved